src/ipyaladin/widget.py

```diff
@@ -220,8 +221,13 @@ def __init__(self, *args: any, **init_options: any) -> None:
         # https://github.com/jupyter-widgets/ipywidgets/blob/main/python/ipywidgets/ipywidgets/widgets/domwidget.py
         for key in ["layout", "tabbable", "tooltip"]:
             init_options.pop(key, None)
+        if "north_pole_orientation" not in init_options:
+            init_options["north_pole_orientation"] = init_options.get("rotation", 0)
+        else:
+            init_options["rotation"] = init_options["north_pole_orientation"]
```

Comment:
nit: This can be simplified if we treat `north_pole_orientation` as the default value and `rotation` as a helper for users, matching what is done in `aladin-lite`. 

This can become the slightly complicated line below
```
init_options["north_pole_orientation"] = init_options.get(
    "north_pole_orientation",
    init_options.get("rotation", 0)
)
```
and then line `230` becomes the following
```
self.rotation = init_options.get("north_pole_orientation", self._rotation)
```